nuget.config

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <config>
+    <add key="repositoryPath" value="packages" />
+  </config>
+  <packageSources>
+    <!-- When <clear /> is present, previously defined sources are ignored -->
+    <clear />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+    <add key="Hl7.Fhir" value="https://www.myget.org/F/fhir-net-api/api/v3/index.json" />
```

Comment:
this feed is for which nuget package?

Reply:
The hl7 packages. This config file is to resolve the restore issue in CI environment

Reply:
I have removed this reference as hl7 packages are already present in nuget.org